.github/pull_request_template.txt

```diff
@@ -0,0 +1,11 @@
+Brief Description
```

Comment:
```suggestion
# Brief Description
```

Reply:
Actually this might be wrong. Any reason why we're using a .txt rather than a .md?

Reply:
The PR template can be either a `.txt` or a `.md` file, it's just that when it's a markdown file, the markdown linter complains about it and I don't necessarily want to conform to the markdown liner haha

Reply:
Hmm, actually having it be an `.md` should be fine.